egs/multi_lang/s5/local/make_corpus_subset.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
Is this different from the babel script? If it is you should mention what the differences are, otherwise make it a soft link to babel script.